client/layout/guided-tours/tracked.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { Component, createElement } from 'react';
+import storeShape from 'react-redux/lib/utils/storeShape';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { GUIDED_TOUR_TRACKED_CLICK } from 'state/action-types.js';
+
+export default (
+		mapPropsToAction = defaultMapPropsToAction
```

Comment:
Should this instead be named something like `selectProps`, `selectedTrackedProps`? We're not actually mapping to an action, as the component itself will later add `type`.